### ISSUE TITLE
sonar-issues-sync FAILS with error - AttributeError: type object 'super' has no attribute 'strictly_identical_to'

### ISSUE THREAD

**Issue by yossile77** [okorach/sonar-tools#916]
Hi,

for long time trying to use that sonar-issues-sync without any success.

This time - that is the command I am using - 

./sonar-issues-sync -k 'management-v1' -b master -K 'management-v1' -B 'release-xxx' --login admin -v DEBUG

[sonarqube 8.9.6]

ngs.py", line 317, in search_siblings
    if finding.strictly_identical_to(self, ignore_component, **kwargs):
  File "/sonarhome/inf/sonar/.local/lib/python3.6/site-packages/sonar/findings/issues.py", line 365, in strictly_identical_to
    return super.strictly_identical_to(another_finding, ignore_component, **kwargs) and (self.debt() == another_finding.debt())
AttributeError: type object 'super' has no attribute 'strictly_identical_to'
bash-4.4

will appreciate a fix for this + if you can try it locally before publishing it. 
if you'd like me to test it prior  I will be happyy to.

Thanks alot!!!

**Comment by okorach** [comment]
Hello @yossile77,
Sorry to read this. I know that `sonar-issues-sync` could be more robust but unfornately I have no means to test anymore since I don't have access to a SonarQube Developer Edition (or Enterprise) anymore.
I will need you assistance to confirm the fix.
The trace you sent should nevertheless allow me to pinpoint the problem.

Note that you're using [SonarQube 8.9, which is now End Of Life](https://community.sonarsource.com/t/when-sonarqube-support-on-8-9-will-end/83343) by Sonar (Now 9.9 and higher only are supported), and I follow the same principle.
The tools may still work with older versions but I don't guarantee it.

**Comment by yossile77** [comment]
Sure  I am here to help with any test required.

Note also it would work with community edition of sonarqube with the sonar branch plugin [https://github.com/mc1arke/sonarqube-community-branch-plugin] - so I think the restriction you put for community edition can be removed as it would work also with sonarqube community edition + community sonar branch plugin.

**Comment by okorach** [comment]
Hello @yossile77,

The problem should be fixed. Can you verify on your platform with the below beta version.
- Download and unzip sonar_tools-2.8.1-py3-none-any.whl.gz
- Install the .whl file.
```
gunzip sonar_tools-2.8.1-py3-none-any.whl.gz
pip install sonar_tools-2.8.1-py3-none-any.whl
```
If you're on window, use a tool that can unzip **.gz** files instead of `gunzip`. The `pip` part remains the same
[sonar_tools-2.8.1-py3-none-any.whl.gz](https://github.com/okorach/sonar-tools/files/13792950/sonar_tools-2.8.1-py3-none-any.whl.gz)

**Comment by okorach** [comment]
Fixed in sonar-tools 2.8.1 just released
